src/update-handling/FileService.ts

```diff
@@ -56,8 +56,9 @@ export class FileService {
             const content = fse.readFileSync(hotkeyFile, 'utf8').toString().split("\n");
             let currentHotkey = {} as RaceHotKey;
             content.forEach((l: string) => {
+                const lowerCaseLine = l.toLowerCase();
                 if (l.startsWith("[")) {
-                    if (currentHotkey.hotKey && currentHotkey.hotkeyCommand) {
+                    if ((currentHotkey.hotKey && currentHotkey.hotkeyCommand) || currentHotkey.researchHotkey) {
```

Comment:
dont knot about that either, is this fine?

Reply:
Yes, all auras are considered "Research", for example, therefore don't have a "Hotkey" but a "Researchhotkey" instead.